plugins/woocommerce/templates/loop/add-to-cart.php

```diff
@@ -12,7 +12,7 @@
  *
  * @see         https://woocommerce.com/document/template-structure/
  * @package     WooCommerce\Templates
- * @version     3.3.0
+ * @version     3.3.1
```

Comment:
I thought this was all that is required for version bump. Why would it expect 8.9.0? Is the version related to Woo rather than a version for the template?

Reply:
It looks that it depends by Woo version:  https://github.com/woocommerce/woocommerce/pull/33681

Reply:
Thanks will fix.